department-of-veterans-affairs/va.gov-cms
# VACMS-2717: Remove unsupported vid options.

## Description
See #2717 

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/92508161-28eae700-f1d6-11ea-8dff-22e74ed28235.png)

## QA steps
As an admin, go to admin/structure/media/manage/video/fields/media.video.field_media_video_embed_field and visually verify that Vimeo and youtube playlist are unchecked.
